frontend/app.json

```diff
@@ -34,6 +36,12 @@
     },
     "web": {
       "favicon": "./src/assets/favicon.png"
-    }
+    },
+    "extra": {
+      "eas": {
+        "projectId": "6fdddfb2-ae12-471d-ac8a-feaff7a00ee5"
+      }
+    },
+    "owner": "campusbuddy"
```

Comment:
Do we want to push this in? We are going to have to get AJ and Johnny set up on Expo. Probably good to do it now rather than later anyways

Reply:
Well if we have it in, whoever wants to run the app will have to run the  "eas login" command first, otherwise the app won't run